src/test/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackServiceTest.java

```diff
@@ -139,10 +144,43 @@ public void shouldTrackCallbackEventOnException() throws Exception {
 
     }
 
+    @Test
+    @DisplayName("Should LogAll callback event")
+    public void shouldLogAllCallbackEvent() throws Exception {
+        List<String> ccdCallbackLogControl = new ArrayList<String>();
+        ccdCallbackLogControl.add("*");
+        doReturn(ccdCallbackLogControl).when(applicationParams).getCcdCallbackLogControl();
+        callbackService.send(URL, CALLBACK_TYPE, caseEventDefinition, null, caseDetails, (Boolean)null);
+
+        verify(appinsights).trackCallbackEvent(eq(CALLBACK_TYPE), eq(URL), eq("200"), any(Duration.class));
+    }
+
+    @Test
+    @DisplayName("Should Log callback event")
+    public void shouldLogCallbackEvent() throws Exception {
+        List<String> ccdCallbackLogControl = new ArrayList<String>();
+        ccdCallbackLogControl.add("test-callback");
+        doReturn(ccdCallbackLogControl).when(applicationParams).getCcdCallbackLogControl();
+        callbackService.send(URL, CALLBACK_TYPE, caseEventDefinition, null, caseDetails, (Boolean)null);
+
+        verify(appinsights).trackCallbackEvent(eq(CALLBACK_TYPE), eq(URL), eq("200"), any(Duration.class));
+    }
+
+    @Test
+    @DisplayName("Should Log callback event")
+    public void shouldNotLogCallbackEvent() throws Exception {
+        List<String> ccdCallbackLogControl = new ArrayList<String>();
+        ccdCallbackLogControl.add("Notest-callback");
+        doReturn(ccdCallbackLogControl).when(applicationParams).getCcdCallbackLogControl();
+        callbackService.send(URL, CALLBACK_TYPE, caseEventDefinition, null, caseDetails, (Boolean)null);
+
+        verify(appinsights).trackCallbackEvent(eq(CALLBACK_TYPE), eq(URL), eq("200"), any(Duration.class));
+    }
+
     private void initSecurityContext() {
         doReturn(principal).when(authentication).getPrincipal();
         doReturn(authentication).when(securityContext).getAuthentication();
```

Comment:
perhaps a test where we test the comma separated config?
e.g.
```
ccdCallbackLogControl.add("callback1, callback2, callback3");
```

Reply:
Not really, Spring will split the comma separated values..
i believe you mean ccdCallbackLogControl.add(callback1).add(callback2) etc

done.

Reply:
I meant testing logging of multiple callbacks. That's done now in `shouldLogCallbackEventMultiple()`